clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/services/ContainerInformationService.java

```diff
@@ -138,13 +153,20 @@ public boolean taskHasHealthCheck(Service service, String accountName, String re
   }
 
   private String toPlatformHealthState(
```

Comment:
Might not be a bad idea to "document" this:
* IF ecs TASK health fails, we fail regardless of other checks
* Then if it's got LB cehcks, return the state of those checks.
* Finally fallback to ECS task health for final confirmation
With the ONLY question being if the user in a pipeline says "ignore LB health checks" which I can't recall the path to get to...

Reply:
NOT a requirement to document this ;) just a "It'd be nice to make sure the logic is TRULY expressed" or more verbose.  BUT I think this overall covers things! MINUS that "how do you disable LB Health checks" call..